airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionItemPage.tsx

```diff
@@ -17,30 +18,10 @@ import { ConnectionSettingsTab } from "./ConnectionSettingsTab";
 import { ConnectionStatusTab } from "./ConnectionStatusTab";
 import { ConnectionTransformationTab } from "./ConnectionTransformationTab";
 
-export const ConnectionItemPage: React.FC = () => {
-  const params = useParams<{
-    connectionId: string;
-    "*": ConnectionSettingsRoutes;
-  }>();
-  const connectionId = params.connectionId || "";
-  const currentStep = params["*"] || ConnectionSettingsRoutes.STATUS;
-  const connection = useGetConnection(connectionId);
-  const [isStatusUpdating, setStatusUpdating] = useState(false);
-  const analyticsService = useAnalyticsService();
+export const ConnectionItemPageInner: React.FC = () => {
+  const { connection } = useConnectionEditService();
```

Comment:
Is using the ConnectionEdit service actually necessary here? Since this is just getting the connection object, couldn't it simply call the `useGetConnection` hook?

Reply:
We want all our components to reference the same connection object out of the connection edit service so they're all up to date.